pkgs/servers/http/nginx/modules.nix

```diff
@@ -47,12 +47,12 @@
     '';
   };
 
-  modsecurity-beta = {
+  modsecurity-nginx = {
```

Comment:
the stable version above is called `modsecurity`. Renaming this one to `modsecurity-nginx` will lead to confusion.

Reply:
Ok. fixed commit.